### PR TITLE
Update vscode module

### PR DESCRIPTION
**Pull request by niku**
# Motivation/Why

I tried to improve vscode-elixir-ls and thought it might be a good to update libraries/dependencies at the first. Because it is obvious to make better which means ease to contribute.

# What

[`vscode`](https://www.npmjs.com/package/vscode) module.

# How

According to [vscode description](https://www.npmjs.com/package/vscode)

> The funcionality of vscode module has been splitted into @types/vscode and vscode-test

so it is not only updating but also splitting to @types/vscode and vscode-test by following instructions in "[Migrating from vscode](https://code.visualstudio.com/api/working-with-extensions/testing-extension#migrating-from-vscode)".

# Why test directory was moved from `/test` to `/src/test`

It was followed by yeoman code generator.

```
$ yo code
? What type of extension do you want to create? New Extension (TypeScript)
? What's the name of your extension? my-sample
? What's the identifier of your extension? my-sample
? What's the description of your extension?
? Initialize a git repository? No
? Which package manager to use? npm

$ tree my-sample/src
my-sample/src
├── extension.ts
└── test
    ├── runTest.ts
    └── suite
        ├── extension.test.ts
        └── index.ts
```

### CODE REVIEW

**Comment by niku** [comment]
Sorry for the poor description. I've updated it.

File: src/test/runTest.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    const extensionDevelopmentPath = path.resolve(__dirname, '../../');
+
+    // The path to test runner
+    // Passed to --extensionTestsPath
+    const extensionTestsPath = path.resolve(__dirname, './suite/index');
+
+    // Download VS Code, unzip it and run the integration test
+    await runTests({ extensionDevelopmentPath, extensionTestsPath });
+  } catch (err) {
+    console.error('Failed to run tests');
```
Comment (axelson) [inline src/test/runTest.ts]:
```suggestion
    console.error(err);
    console.error('Failed to run tests');
```

Can we add this back in from: https://code.visualstudio.com/api/working-with-extensions/testing-extension#the-test-script

It seems like printing the error would be useful when trying to debug a failing test suite.

**Comment by niku** [comment]
Thank you for your suggestion. I've applied it.

**Comment by axelson** [comment]
Hmmm, I should have squashed the commits together when merging.

**Pull request merged**